src/components/Header/AuthMenu.js

```diff
@@ -4,11 +4,21 @@ import { Link } from 'react-router-dom';
 import { GoKebabVertical, GoX } from 'react-icons/go';
 // styles
 import styles from './header.module.scss';
+import Signin from '../../containers/SignIn/SignIn'
 
 class AuthMenu extends Component {
   constructor(props) {
     super(props);
     this.triggerMobileCategory = this.triggerMobileCategory.bind(this);
+
+    this.state = {
+      showModal: false
+    }
+  }
+
+  toggleModal = () => {
+    const { showModal } = this.state;
+    this.setState({ showModal: !showModal })
   }
 
   triggerMobileCategory() {
```

Comment:
is there any need for this method?

Reply:
Yes, this is the method that toggles the sign in modal component's visibility. It is tied to the button on the header.

Reply:
I actually meant this `triggerMobileCategory`. I can see it's empty